CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Mapbox welcomes participation and contributions from everyone.
 ## Bug fixes 🐞
 * Fix scale bar truncated at high zoom levels near the poles. ([1620](https://github.com/mapbox/mapbox-maps-android/pull/1620))
 
+# 10.9.0-beta.1
+## Features ✨ and improvements 🏁
+* Remove redundant exception handling for `CameraManager.setCamera` method. Exception in `CameraManager.setCamera` now handled in native.([1624](https://github.com/mapbox/mapbox-maps-android/pull/1624))
```

Comment:
We did remove redundant exception handling in _camera animation plugin_ and not in camera manager. Moreover if someone was not using for some reason our animation system and was calling `MapboxMap#setCamera` directly - now they could be sure that this method is exception free.